### ISSUE TITLE
vlan show with vlan filter returns empty string

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#262]
**Describe the bug**
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli -c ~/configs/multidc.yml
jpietsch> vlan unique columns=vlan
   vlan  count
3     1      8
4    10      8
2  4001     12
0    24     36
1    13     36
jpietsch> vlan show vlan=24
Empty DataFrame
Columns: [namespace, hostname, ifname, vlan, pvid, timestamp]
Index: []
jpietsch> 
```

**Comment by jopietsch** [comment]
this looks fixed, is it fixed @ddutt ?

**Comment by ddutt** [comment]
Yes,this is closed.

**Comment by ddutt** [comment]
Fixed by 94f7193eb2fb04691702307fbccd7b8fe3589afa and 47e8ec7e3bc7d8e2fb8e05b865b8e8c79286bf8d
